policy/modules/services/xserver.if

```diff
@@ -367,7 +367,7 @@ template(`xserver_common_x_domain_template',`
 		attribute input_xevent_type;
 
 		class x_drawable all_x_drawable_perms;
-		class x_property all_x_property_perms;
+		#class x_property all_x_property_perms;
```

Comment:
Please remove the line instead of commenting.

Reply:
I commented because the potential usage is also commented out:
https://github.com/SELinuxProject/refpolicy/blob/45f43ca3782065c72db3414ba7add6351ca66dcf/policy/modules/services/xserver.if#L386
Should this line also be removed?

Reply:
No, as the permissions wouldn't be used in the `type_transition`.

Reply:
But the class is (after fixing the missing type `xproperty_t`):

```
/usr/bin/checkmodule -m tmp/bluetooth.tmp -o tmp/bluetooth.mod
policy/modules/services/bluetooth.te:219:ERROR 'unknown class x_property' at token ';' on line 8685:
        type_transition bluetooth_helper_t xproperty_t:x_property bluetooth_helper_xproperty_t;
#line 219
/usr/bin/checkmodule:  error(s) encountered while parsing configuration
```

Reply:
Hmm, I didn't expect that. In light of that, lets keep what you have.